pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -94,7 +94,7 @@ self: super: builtins.intersectAttrs super {
   # Won't find it's header files without help.
   sfml-audio = appendConfigureFlag super.sfml-audio "--extra-include-dirs=${pkgs.openal}/include/AL";
 
-  cachix = enableSeparateBinOutput super.cachix;
+  cachix = enableSeparateBinOutput (appendPatch super.cachix ./patches/cachix-servant-0.16.2-compat.patch);
```

Comment:
Is there anywhere to get this from upstream?

In general it seems like people don't like carrying patches around in the nixpkgs repo if it is possible to just `fetchpatch` them from some upstream repo.  If you grep for `fetchpatch` in nixpkgs, you find a lot of places doing this.

Of course, if it is not possible, then carrying the patch here is the only realistic option.

Reply:
sure, I've replace it with `fetchpatch`